config/default.json

```diff
@@ -70,6 +70,7 @@
         "EnableTeamCreation": true,
         "EnableUserCreation": true,
         "EnableOpenServer": false,
+        "EnableUserDeactivation": false,
```

Comment:
This is not used anywhere. I would expect to check that setting at updateUserActive.

Reply:
Maybe in the future it could be a permission in the permissions system (I don't know)

Reply:
we are using that on the client side to enable/disable the menu to the user deactivate himself. I will use that in the app as well to make sure the user doesn't call the service if the flag is disabled.

Reply:
did the change. PTAL @jespino thanks for your review